browser/src/map/handler/Map.Settings.ts

```diff
@@ -0,0 +1,136 @@
+/* -*- js-indent-level: 8 -*- */
+/*
+ * Copyright the Collabora Online contributors.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+/*
+ * L.Map.Settings.
+ */
+
+interface IFrameDialog {
+	remove(): void;
+	hasLoaded(): boolean;
+	postMessage(message: any): void;
+	show(): void;
+}
+
+L.Map.mergeOptions({
+	settings: true,
+});
+
+L.Map.Settings = L.Handler.extend({
```

Comment:
It's "Leaflet style" class. It's our old dependency. Now we migrate to standard ES6 classes.
Can be converted later as similar dialogs exists so we will do some base class possibly. Not to be done in this PR, just FYI.